PoseEstimation-TFLiteSwift/PoseEstimator.swift

```diff
@@ -76,6 +76,16 @@ struct PoseEstimationOutput {
     typealias Line = (from: Keypoint, to: Keypoint)
     var keypoints: [Keypoint] = []
     var lines: [Line] = []
+    
+    func filterKeypoints(with threshold: CGFloat?) -> [Keypoint] {
```

Comment:
I think, 'filtered' more appropriate than 'filter'.
Because, This function don't return a filter.